### PR TITLE
Cosmetics/general page design

### PR DESCRIPTION
**Pull request by agfagerbakk**
# Changes 🈂️  🎨 🖌 
Changing general project layout
* Add project-wide general fonts, color schemes, remove reduntant code within said change

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,9 +20,10 @@
     "@angular/platform-browser": "~7.0.0",
     "@angular/platform-browser-dynamic": "~7.0.0",
     "@angular/router": "~7.0.0",
+    "ajv": "^6.10.0",
```
Comment (jonjohansen) [inline package.json]: What does this do?

File: src/app/todo-card/todo-card.component.html [hunk 2]
```diff
@@ -1,6 +1,5 @@
 <div class="todo-card">
   <h1 class="card-title">{{ todo.title }}</h1>
-  <p class="card-description">{{ todo.description }}</p>
```
Comment (jonjohansen) [inline src/app/todo-card/todo-card.component.html]: A description is completely removed, is this intentional?

**Review by jonjohansen — CHANGES_REQUESTED** [review]

File: src/app/todo-card/todo-card.component.html (hunk 2 above)
Comment (agfagerbakk) [reply src/app/todo-card/todo-card.component.html]: Yeah, it is redundant

File: package.json (hunk 1 above)
Comment (agfagerbakk) [reply package.json]: Removed that

**Review by jonjohansen — DISMISSED** [review]
Looks good to me 💯

**Review by markhilb — APPROVED** [review]
Nice job 😃

**Pull request merged**